stdlib/private/StdlibUnittest/StdlibUnittest.swift.gyb

```diff
@@ -1232,6 +1234,9 @@ public enum TestRunPredicate : CustomStringConvertible {
 
     case FreeBSDAny(reason: let reason):
       return "FreeBSDAny(*, reason: \(reason))"
+
+    case noObjC(let reason):
```

Comment:
I would phrase it avoiding the negation: `case ObjCRuntime`, `case NativeRuntime`.  What do you think?